src/main/scala/com/github/symcal/Expr.scala

```diff
@@ -1,37 +1,97 @@
 package com.github.symcal
 
+import scala.language.implicitConversions
```

Comment:
IntelliJ gives a warning unless this is included. Maybe this is somehow important for the compiler.